lib/coophub/backends/backends.ex

```diff
@@ -13,6 +13,21 @@ defmodule Coophub.Backends do
   @type langs :: %{String.t() => integer()}
   @type topics :: [String.t()]
 
+  @callback name() :: String.t()
+
+  @callback prepare_request_org(String.t()) :: request_data
+  @callback parse_org(map) :: org
+
+  @callback prepare_request_repos(org, integer) :: request_data
+  @callback prepare_request_repo(org, map) :: request_data
+  @callback parse_repo(map) :: repo
+
+  @callback prepare_request_topics(org, repo) :: request_data
+  @callback parse_topics(any) :: topics
+
+  @callback prepare_request_languages(org, repo) :: request_data
+  @callback parse_languages(any) :: langs
```

Comment:
The `@callback prepare_request_rate_limit` isn't missing here?

Reply:
Not needed because the commented here https://github.com/fiqus/coophub/pull/90/files/bf80d1acb6acf43bef8fc12fec858cca38ec7f01#r664858348

But let me check if it could be placed directly in the github backend